SchedulerJobs/SchedulerJobs/Functions/AnonymiseHearingsConferencesAndDeleteAadUsersFunction.cs

```diff
@@ -0,0 +1,32 @@
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SchedulerJobs.Services;
+using System;
+using System.Threading.Tasks;
+using Willezone.Azure.WebJobs.Extensions.DependencyInjection;
+
+namespace SchedulerJobs.Functions
+{
+    public static class AnonymiseHearingsConferencesAndDeleteAadUsersFunction
+    {
+        /// <summary>
+        /// Function to anonymise hearing and conference data older than three months
+        /// </summary>
+        /// <param name="myTimer">Set time to run every day at 1:10 AM</param>
+        /// <param name="log"></param>
+        /// <param name="anonymiseHearingsConferencesDataService"></param>
+        [FunctionName("AnonymiseHearingsConferencesAndDeleteAadUsersFunction")]
+        public static async Task Run([TimerTrigger("* */5 * * * *")]TimerInfo myTimer, 
```

Comment:
Timer will fire every 5 mins.. is that the correct time... just checking with you?

Reply:
Agreed 5.30 AM every night. Once only

Reply:
updated to 5:30 am